src/lib/prov/tpm2/tpm2_crypto_backend/tpm2_crypto_backend.cpp

```diff
@@ -566,16 +573,66 @@ static TSS2_RC get_ecdh_point(TPM2B_PUBLIC* key,
                               BYTE* out_buffer,
                               size_t* out_size,
                               void* userdata) {
+   #if defined(BOTAN_HAS_TPM2_ECC_ADAPTER)
+   return thunk([&] {
```

Comment:
Explicitly set `out_size = 0` at the beginning of this. So if anything goes wrong, we can be sure that this out param is in a "saf" state

Reply:
Added to `rsa_pk_encrypt` as well